static/js/forum.js

```diff
@@ -216,7 +216,7 @@
     
     $.forum.bootstrap = function() {
         $.forum.app = new $.forum.Router(); 
-        Backbone.history.start({pushState: true});
+        Backbone.history.start();
```

Comment:
It is quickfix - routing urls like /thread/<id> dont work with pushstate 
without server-side code. But #thread/<id> do work.

20-Jul-16 4:04 PM, Xavier Antoviaque пишет:
>
> In static/js/forum.js 
> <https://github.com/antoviaque/backbone-relational-tutorial/pull/9#discussion_r71519747>:
>
> > @@ -216,7 +216,7 @@
> >
> >      $.forum.bootstrap = function() {
> >          $.forum.app = new $.forum.Router();
> > -        Backbone.history.start({pushState: true});
> > +        Backbone.history.start();
>
> This is now the default?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/antoviaque/backbone-relational-tutorial/pull/9/files/612ed2770ff41dc41fc79f869cda367bb61e9250#r71519747>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ANjyUXCeIT1EIzRfJnzHKw4iF69ex3lKks5qXhzigaJpZM4JPupt>.
>